addons/mrp_subcontracting_dropshipping/tests/test_anglo_saxon_valuation.py

```diff
@@ -205,3 +205,61 @@ def test_avco_valuation_subcontract_and_dropshipped_and_backorder(self):
                 {'reference': dropship_backorder.name, 'quantity': 0, 'value': -8000},
             ]
         )
+
+    def test_account_line_entry_kit_bom_dropship(self):
+        """ An order delivered via dropship for some kit bom product variant should result in
+        accurate journal entries in the expense and stock output accounts if the cost on the
+        purchase order line has been manually edited.
+        """
+        kit_final_prod = self.product_a
+        kit_bom = self.env['mrp.bom'].create({
+            'product_tmpl_id': kit_final_prod.product_tmpl_id.id,
+            'product_uom_id': kit_final_prod.uom_id.id,
+            'product_qty': 1.0,
+            'type': 'phantom',
+        })
+        kit_bom.bom_line_ids = [(0, 0, {
+            'product_id': self.product_b.id,
+            'product_qty': 1,
+        })]
+
+        self.env['product.supplierinfo'].create({
+            'product_id': self.product_b.id,
+            'partner_id': self.partner_a.id,
+            'price': 2000,
+        })
+
+        (kit_final_prod + self.product_b).categ_id.write({
+            'property_cost_method': 'fifo',
+            'property_valuation': 'real_time',
+        })
+
+        sale_order = self.env['sale.order'].create({
+            'partner_id': self.partner_b.id,
+            'order_line': [(0, 0, {
+                'product_id': kit_final_prod.id,
+                'route_id': self.dropship_route.id,
+                'product_uom_qty': 2.0,
```

Comment:
force the sale price unit to 900 as it seems it's changed in an independent module making this test failing at install